### ISSUE TITLE
A11Y color contrast

### ISSUE THREAD

**Issue by fgeierst** [Xippo/OM-Cookie-Manager#23]
Lighthouse accessibility test claims the color contrast of the button with white text on green background is not sufficient. Even with the upcoming more fine grained APAC metric, it's failing contrast requirements. 
A solution would be to switch to a darker green.

<img width="536" alt="image" src="https://user-images.githubusercontent.com/22551442/206669424-204ea41f-72f2-4032-a9b8-ebf4be83aefd.png">
<img width="944" alt="image" src="https://user-images.githubusercontent.com/22551442/206669500-3588b01e-9a30-4faf-b2fe-5eb4ff0e7861.png">

**Comment by Xippo** [comment]
Color is changed the contrast is know 4.5+

**Comment by fgeierst** [comment]
Nice, thanks!
